### ISSUE TITLE
whatevergreen_kext issues

### ISSUE THREAD

**Issue by niyan-ly** [sarkrui/NUC8i7BEH#7]
configuration:
- nuc: nuc8i5BEH
- ram: hyperX Impact 8G*2
- disk: crucial mx500 500GB

It seems like that if there is no eGPU being used, then the whatever green kext will result an unpredictable reboot(well, do predictable with some specific actions like launch an iOS simulator). I did some research, and i found that whatever_green kext is usually used for sky lake and Stand alone graphics card.

In conclusion, i finally resolved the random reboot issue after removing the whatgreen kext.
